fpgalib/dac.py

```diff
@@ -83,6 +83,46 @@
 # TODO
 # Think of better variable names than self.params and self.boardParams
 
+# TODO: Integrate this with DAC class
+# TODO: Have dac_calibration_server not DACify SRAM data
+def waves2sram(wave_a, wave_b, trigger_idx=(16, 17)):
```

Comment:
I'd rather call this `dacify`, and in either case we should take this opportunity to fix the name (i.e. no camelCase).